libs/SalesforceReact/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ plugins {
 
 dependencies {
     api(project(":libs:MobileSync"))
-    api("com.facebook.react:react-native:0.70.14")
+    api("com.facebook.react:react-android:0.72.6")
```

Comment:
So you are doing Gradle 8 and React Native upgrade in one shot?
You will need a parallel PR on the SalesforceMobileSDK-ReactNative repo then.

Reply:
Correct.  If I recall the React Native documentation, 0.72 was their move to Gradle 8 and AGP 8.  I'll take a look at `SalesforceMobileSDK-ReactNative`. Thanks!

Reply:
Though it's currently work in progress, 👉🏻 https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative/pull/345 is open.